frontend/awx/main/routes/useAwxInventoryRoutes.tsx

```diff
@@ -119,6 +121,11 @@ export function useAwxInventoryRoutes() {
           path: ':inventory_type/:id/host/:host_id/edit',
           element: <EditHost />,
         },
+        {
+          id: AwxRoute.InventoryGroupRelatedGroupsCreate,
+          path: ':inventory_type/:id/groups/:group_id/related_groups/add',
```

Comment:
Small change to match the route specified in [AAP-8314](https://issues.redhat.com/browse/AAP-8314) and the old UI
```suggestion
          path: ':inventory_type/:id/groups/:group_id/nested_groups/add',
```